sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -50,14 +50,14 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit a76e3b185a5e1df9114dc483077227f88f522244
-  # Merge: 3653e582 657ffde9
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Fri Oct 20 12:12:15 2023 +0100
-  #     Merge pull request #972 from oneapi-src/revert-906-l0_usm_error_checking
-  #     Revert "[UR][L0] Propagate errors from `USMAllocationMakeResident`"
-  set(UNIFIED_RUNTIME_TAG a76e3b185a5e1df9114dc483077227f88f522244)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/bensuo/unified-runtime.git")
+  #commit 9a13afd59a830a19a0d337bd3197c32ea3880aa8
+  #Merge: TODO
+  #Author: Ewan Crawford <ewan@codeplay.com>
+  #Date:   Mon Oct 16 16:26:44 2023 +0100
+  #        TODO
+  #        [Command-buffer][L0] Reset sync-point events
+  set(UNIFIED_RUNTIME_TAG 9a13afd59a830a19a0d337bd3197c32ea3880aa8)
```

Now that https://github.com/oneapi-src/unified-runtime/pull/965 is merged please pull in the latest `sycl` branch and update the `UNIFIED_RUNTIME_TAG` as follows:

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 7c8617b3710736db642585fe310a64e19e059d2e
  # Merge: 20d4f49 9a13afd
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Thu Oct 26 10:20:03 2023 +0100
  #     Merge pull request #965 from Bensuo/ewan/L0_internal_event_fix
  #     [Command-buffer][L0] Reset sync-point events
  set(UNIFIED_RUNTIME_TAG 7c8617b3710736db642585fe310a64e19e059d2e)
```